src/functions/durableOrchestration.ts

```diff
@@ -0,0 +1,25 @@
+import { HttpRequest, InvocationContext } from '@azure/functions';
+import * as df from 'durable-functions';
+import { DurableOrchestrationClient, IOrchestrationFunctionContext } from 'durable-functions/lib/src/classes';
+
+df.orchestration('DurableFunctionsOrchestratorJS', function* (context: IOrchestrationFunctionContext) {
+    const outputs = [];
+    outputs.push(yield context.df.callActivity('Hello', 'Tokyo'));
+    outputs.push(yield context.df.callActivity('Hello', 'Seattle'));
+    outputs.push(yield context.df.callActivity('Hello', 'Cairo'));
+
+    return outputs;
+});
+
+df.activity('Hello', (context: InvocationContext, name: string) => {
+    return `Hello, ${name}`;
+});
+
+df.httpClient(
+    'DurableFunctionsHttpStart',
+    async (context: InvocationContext, req: HttpRequest, client: DurableOrchestrationClient) => {
+        const instanceId = await client.startNew(req.query.get('functionName'), undefined, req.body);
```

Comment:
`req.body` has a different type than the old model so I don't think it'll work. You'll probably have to do `req.text()` or `req.arrayBuffer()`

Reply:
Fixed